assets/src/models/blockWaiver.ts

```diff
@@ -32,25 +35,26 @@ export const hasCurrentBlockWaiver = ({
 }: VehicleOrGhost): boolean =>
   blockWaivers.some(
     (blockWaiver) =>
-      currentFuturePastType(blockWaiver) === CurrentFuturePastType.Current
+      currentFuturePastType(blockWaiver, 240) === CurrentFuturePastType.Current
   )
 
 /**
- * has waiver?      | ghost       | vehicle and late indicator ghosts
- * ---------------- | ----------- | ---------------------------------
- * yes, current     | black       | black
- * yes, not current | highlighted | grey
- * none             | highlighted | none
+ * has waiver?              | ghost       | late ghost | vehicle
+ * ------------------------ | ----------- | ---------- | -------
+ * yes, current or soon     | black       | black      | black
+ * yes, not current         | highlighted | none       | none
+ * none                     | highlighted | none       | none
  */
 export const blockWaiverAlertStyle = (
   vehicleOrGhost: VehicleOrGhost
 ): AlertIconStyle | undefined => {
   if (hasCurrentBlockWaiver(vehicleOrGhost)) {
     return AlertIconStyle.Black
   }
+
   if (isGhost(vehicleOrGhost) && !isLateVehicleIndicator(vehicleOrGhost)) {
     return AlertIconStyle.Highlighted
-  } else {
-    return hasBlockWaiver(vehicleOrGhost) ? AlertIconStyle.Grey : undefined
```

Comment:
Since we no longer have grey icons anymore, that value can be removed from the enum in `iconAlertCircle.tsx` and from the scss in `_icon_alert_circle.scss`